core/platform/taskqueue/dev_mode_taskqueue_services.py

```diff
@@ -39,7 +47,7 @@ def _task_handler(url, payload, queue_name, task_name=None):
         queue_name: str. The name of the queue to add the task to.
         task_name: str|None. Optional. The name of the task.
     """
-    headers = {}
+    headers: Dict[str, Optional[str]] = {}
```

Comment:
Why `Optional`?

Reply:
Because task_name [here](https://github.com/oppia/oppia/blob/37c24c46e6fc5a9ee1e8df6fb62d55737fa1ced3/core/platform/taskqueue/dev_mode_taskqueue_services.py#L44) can be None.

Reply:
Hmmm, lets make it non-optional and put some generic string in the call site.

Reply:
Changed the type to `str` and default value from `None` to empty string.

Reply:
Done